### PR TITLE
feat: callouts

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-9011 |
| :--------------------: | :---------: |

## 🧰 Changes

Gets markdown callouts working!

For testing, I did a big refactor on the demo(?) app. I replaced all the changing params stuff with `react-router-dom`, and updated it to use typescript!

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-841.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: index.tsx [hunk 1]
```diff
@@ -34,15 +33,19 @@ export const utils = {
...
-  return createProcessor(opts);
+  return createProcessor({ remarkPlugins: [calloutTransformer], ...opts });
 };
 
 export const react = (text: string, opts = {}) => {
   const code = compileSync(text, {
-    development: true,
     outputFormat: 'function-body',
     providerImportSource: '@mdx-js/react',
+    remarkPlugins: [calloutTransformer],
```
Comment (kellyjosephprice) [inline index.tsx:48]: These are the only changes required for getting callouts to work!

**Review by trishaprile — COMMENTED** [review]
yay love the demo refactor and mdx callouts!
although when I try to manually type one out with `<` the page crashes

**Comment by kellyjosephprice** [comment]
> although when I try to manually type one out with `<` the page crashes

Hey, nobodies perfect, ok. I added some error boundaries.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.13</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.13)
<br>

**Pull request merged**